src/subscription_manager/migrate/migrate.py

```diff
@@ -710,7 +710,7 @@ def register(self, credentials, org, environment):
 
         if self.options.service_level:
             servicelevel = self.select_service_level(org, self.options.service_level)
-            cmd.append('--servicelevel=' + servicelevel)
+            cmd.append('--service-level=' + servicelevel)
```

Comment:
The option seems `--servicelevel` actually, see src/subscription_manager/cli_command/register.py:
```python
        self.parser.add_argument("--servicelevel", dest="service_level",
                               help=_("system preference used when subscribing automatically, requires --auto-attach"))
```